daemon/albatrossd.ml

```diff
@@ -145,14 +145,39 @@ let jump _ influx tmpdir dbdir =
   | Error (`Msg msg) -> Logs.err (fun m -> m "bailing out: %s" msg)
   | Ok old_unikernels ->
     Lwt_main.run
-      (let unix_connect s =
-         Vmm_lwt.connect Lwt_unix.PF_UNIX (Lwt_unix.ADDR_UNIX (socket_path s))
+      (let rec unix_connect ~retries s =
+         Vmm_lwt.connect Lwt_unix.PF_UNIX (Lwt_unix.ADDR_UNIX (socket_path s)) >>= fun x ->
```

Comment:
```suggestion
         Vmm_lwt.connect Lwt_unix.PF_UNIX (Lwt_unix.ADDR_UNIX (socket_path s)) >>= function
```

the `fun x -> match x with ..patterns..` can be condensed to `function ..patterns..`